### DISCUSSION TITLE
Welcome to photon Discussions!

### DISCUSSION THREAD

**Discussion by lonvia** [komoot/photon#528] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by brahica** [comment]
Hello @lonvia, 

Nice project.

I wonder if we can search in any language but force the result in a specific language?

Thx you

**Reply by stephan75** [reply]
Can you give us an example?

**Reply by lonvia** [reply]
You can use the [`lang=` parameter](https://github.com/komoot/photon#adjust-language) to choose a language. The photon website and the dump we provide have English(en), German(de) and French(fr) enabled.

NB: better open a new discussion thread for questions like these.

**Reply by brahica** [reply]
Thanks a lot, my bad.
